web/src/main/java/se/inera/intyg/webcert/web/integration/interactions/sendmessagetocare/SendMessageToCareResponderImpl.java

```diff
@@ -51,19 +51,26 @@ public SendMessageToCareResponseType sendMessageToCare(String logicalAddress, Se
             arendeService.processIncomingMessage(ArendeConverter.convert(request));
             result.setResultCode(ResultCodeType.OK);
         } catch (WebCertServiceException e) {
-            result.setResultCode(ResultCodeType.ERROR);
             switch (e.getErrorCode()) {
+                case MESSAGE_ALREADY_EXISTS:
+                    result.setResultCode(ResultCodeType.INFO);
+                    result.setResultText(e.getMessage());
+                    LOG.info("Could not process incoming message to care. Message already exists. {} : {}", e.getErrorCode(),
```

Comment:
Can we include the message id and maybe even the certificate id, to make it easier to find situations when this happens?

Same for the other logs.